### PR TITLE
For #3203: Updates edit bookmarks to have user friendly names

### PR DESCRIPTION
**Pull request by colintheshots**
Co-authored-by: Colin Lee <mncolinlee@gmail.com>
Co-authored-by: Sawyer Blatz <sdblatz@gmail.com>

Sawyer asked for help extracting extension methods from BookmarkFragment for his existing PR. So I created this one as I couldn't push onto his.

### CODE REVIEW

**Comment by sblatz** [comment]
Looks like this is having some issues with bitrise/TC.

**Comment by colintheshots** [comment]
I fixed the test that broke by adding an import statement.

File: app/src/test/java/org/mozilla/fenix/library/bookmarks/BookmarkViewModelTest.kt [hunk 1]
```diff
@@ -14,6 +14,7 @@ import org.junit.Before
 import org.junit.Test
 import org.mozilla.fenix.TestUtils
 import org.mozilla.fenix.TestUtils.bus
+import org.mozilla.fenix.ext.minus
```
Comment (sblatz) [inline app/src/test/java/org/mozilla/fenix/library/bookmarks/BookmarkViewModelTest.kt]: Nit: we can remove this import maybe?

Comment (colintheshots) [reply app/src/test/java/org/mozilla/fenix/library/bookmarks/BookmarkViewModelTest.kt]: It's an extension operator. We'd have to add it to the class to remove the import.

**Pull request merged**